backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidInput.java

```diff
@@ -127,8 +128,7 @@ protected TouchEvent newObject () {
 	final Context context;
 	protected final AndroidTouchHandler touchHandler;
 	private int sleepTime = 0;
-	private boolean catchBack = false;
-	private boolean catchMenu = false;
+	private IntArray catchedKeys = new IntArray();

```

Comment:
Is there any reason why you didn't use an IntSet instead?
It should be faster (`onKey()` may be called many times) than an IntArray and also easier to use in my opinion.

Reply:
Just because I was not aware of it, will change it.